src/workerd/api/tests/worker-test-module-does-eval.js

```diff
@@ -0,0 +1 @@
+eval(`console.log('I am a module that does eval')`);
```

Comment:
As Kenton suggests, we should add asserts to ensure the eval ran. 

Also, would be good to include some `new Function(...)` calls in the tests as well, just to be thorough